packages/syft/src/syft/core/node/common/action/get_object_action.py

```diff
@@ -191,23 +191,6 @@ def execute_action(
 
             msg = GetObjectResponseMessage(obj=obj, address=self.reply_to, msg_id=None)
 
-            if self.delete_obj:
```

Comment:
Doesn't this mean that objects are no longer deleted when a user calls .get()?

Reply:
@madhavajay they are deleted, but not inside the `get_object_action.py`. The logic to delete is moved to a different message. Refer to:
https://github.com/OpenMined/PySyft/pull/6385/files#diff-b52976171f0fc89f259716fc13dd749fdbf453a72a5cfe2be9537e07a68d56feR238

Reply:
Yes, awesome. I can see now there is a seperate step which calls an explicit Delete this is great and much much better! ❤️